Makefile

```diff
@@ -5,4 +5,5 @@ develop:
 	pip install -q -e .
 
 test: develop
+	pip install pytest pytest-django
```

Comment:
These need to go in `requirements.txt`. Then installing them is covered by the `develop` task.